packages/codegen-ui/lib/generate-form-definition/helpers/form-field.ts

```diff
@@ -339,6 +339,7 @@ export function getFormDefinitionInputElement(
 
   formDefinitionElement.validations = mergedValidations;
   formDefinitionElement.dataType = config?.dataType || baseConfig?.dataType;
+  formDefinitionElement.isArray = config.inputType?.isArray || baseConfig?.inputType?.isArray;
 
```

Comment:
I think we want the DataStore `isArray` config to take priority. If a user has a DataStore field that is just a scalar, they shouldn't be able to override that with an array but that might be a @renebrandel question.

Reply:
Similarly for the reverse, if the field is a scalar and the user configures it as an array, does the use have to map it correctly in code?

Reply:
new version prioritizes value in baseConfig if present